docs/root/configuration/http/http_conn_man/headers.rst

```diff
@@ -48,6 +48,13 @@ server
 The *server* header will be set during encoding to the value in the :ref:`server_name
 <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.server_name>` option.
 
+.. _config_http_conn_man_headers_referer:
+
+referer
+-------
+
+The *referer* header will be sanitized during decoding. The feature is guarded by *envoy.reloadable_features.sanitize_http_header_referer* runtime flag.
```

Comment:
The runtime guards are expected to be transient and not to land in docs.  Let's comment what is actually done: multiple referrer URLs or invalid URLs will be removed, and then we can update it as we handle more of the feature